### PR TITLE
refactor(all) remove function prefix from all shell files

### PR DESCRIPTION
**Pull request by brij-desaii**
for issue #464

### CODE REVIEW

**Comment by y0rune** [comment]
Hello!
I added the comments when you remove the word `function`.
The files with `.md` it should not been changed. I think you done a little bit more regex to remove it in the repo.

File: node/install.sh [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 # "This is too simple" you say! "Where is the magic!?" you ask.
 # There is no magic!
-# The custom functions for node are here.
-# The generic functions - version checks, download, extract, etc - are here:
+# The custom s for node are here.
```
Comment (y0rune) [inline node/install.sh:5]: I think it should be restore the word `function` here ;)

File: node/install.sh [hunk 2]
```diff
@@ -2,8 +2,8 @@
...
+# The generic s - version checks, download, extract, etc - are here:
```
Comment (y0rune) [inline node/install.sh:6]: I think it should be restore the word `function` here ;)

**Review by y0rune — CHANGES_REQUESTED** [review]

**Review by y0rune — CHANGES_REQUESTED** [review]
Hello!
I added the comments when you remove the word `function`.
The files:
- `.md` 
- `jshint/jshintrc.defaults.json5`

it should not been changed. 

I think you done a little bit more regex to remove it in the repo.

**Comment by coolaj86** [comment]
@brij-desaii Do you want to try this again?

**Comment by brij-desaii** [comment]
> 



>

**Comment by brij-desaii** [comment]
Hey, @coolaj86! This is my first time contributing to an opensource project on github and I'm a bit confused 😅. As per my understanding, I have accidently removed the word "function" from the comments in the shell files and I'm not sure how to fix this mistake. Could you please help me out?

**Comment by coolaj86** [comment]
@brij-desaii
1. You can't just do a find-and-replace through _all_ of the files. You need to do it only to `.sh` files.
2. You need to use a tool that can pattern match `function (name)\(\)` and replace it with `$1()`. I believe I gave an example of that in the issue with [`sd`](https://webinstall.dev/sd).
3. To fix the git history you can:
  - `git reset main --hard` to undo your old commit
  - `git push --force` to push over your current PR

**Pull request closed without merging**